doc/api/repl.md

```diff
@@ -411,6 +417,8 @@ added: v0.1.91
     `SIGINT` is received, i.e. `Ctrl+C` is pressed. This cannot be used together
     with a custom `eval` function. Defaults to `false`.
 
+If `options` is a string, then it specifies the input prompt.
+
```

Comment:
Well, it is almost documented in some examples. See:

https://github.com/nodejs/node/blame/master/doc/api/repl.md#L101
https://github.com/nodejs/node/blame/master/doc/api/repl.md#L120

If it is not official API, maybe we should change these fragments?